scripts/install_from_release.sh

```diff
@@ -12,13 +12,42 @@ fi
 
 # Map the language parameter to the corresponding scenarios file
 # If no parameter, download the scenarios from IE
-if [ "$LANG" = "" ]; then
+MAIN_LANG_PREFIX="$(echo "$LANG" | head -c2 | tr '[:upper:]' '[:lower:]')"
+LANG_ARRAY_1=("de" "es" "fr" "it" "nl" "pt")
+LANG_ARRAY_2=("cs" "hu" "id" "ja" "ko" "pl" "pt" "ru" "sv" "tr")
+
+if [ "$MAIN_LANG_PREFIX" = "" ] || [ "$MAIN_LANG_PREFIX" = "en" ]; then
   SCENARIOS="https://github.com/Azure/InnovationEngine/releases/download/$RELEASE/scenarios.zip"
-# Otherwise, download the scenarios from Microsoft Docs in the appropriate langauge
-elif [ "$LANG" = "en-us" ]; then
-  SCENARIOS='https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/scenarios.zip'
+elif [ "$MAIN_LANG_PREFIX" = "pt" ]; then
+  if [ "$LANG" = 'pt-pt' ]; then
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/pt-pt-scenarios.zip"
+  elif [ "$LANG" = 'pt-br' ]; then
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/pt-br-scenarios.zip"
+  fi
+elif [ "$MAIN_LANG_PREFIX" = "zh" ]; then
+  if [ "$LANG" = "zh-cn" ]; then
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/zh-cn-scenarios.zip"
+  elif [ "$LANG" = "zh-tw" ]; then 
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/zh-tw-scenarios.zip"
+  else
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/zh-cn-scenarios.zip"
+  fi
+elif [[ ${LANG_ARRAY_1[@]} =~ $MAIN_LANG_PREFIX ]]; then
+  # for any other language that we do not have a specific scenarios file for, we will use the generic one
+  SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/$MAIN_LANG_PREFIX-$MAIN_LANG_PREFIX-scenarios.zip"
+elif [[ ${LANG_ARRAY_2[@]} =~ $MAIN_LANG_PREFIX ]]; then
+  SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/$MAIN_LANG_PREFIX-$MAIN_LANG_PREFIX-scenarios.zip"
+  if [ "$MAIN_LANG_PREFIX" = "cs" ]; then
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/$MAIN_LANG_PREFIX-cz-scenarios.zip"
+  elif [ "$MAIN_LANG_PREFIX" = "ja" ]; then
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/$MAIN_LANG_PREFIX-jp-scenarios.zip"
+  elif [ "$MAIN_LANG_PREFIX" = "ko" ]; then
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/$MAIN_LANG_PREFIX-kr-scenarios.zip"
+  elif [ "$MAIN_LANG_PREFIX" = "sv" ]; then
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/$MAIN_LANG_PREFIX-se-scenarios.zip"
+  fi
 else
-  SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/$LANG-scenarios.zip"
+  SCENARIOS="https://github.com/Azure/InnovationEngine/releases/download/$RELEASE/scenarios.zip"
```

Comment:
I feel like we could move this last else to the beginning so it runs always (and remove 19-20).  Then if any of the if statements happen it'll just overwrite SCENARIOS.
Also in the default/english case, we're not downloading scenarios from IE anymore:
```suggestion
  SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/scenarios.zip"
```